### PR TITLE
Script source/publish location fix

### PR DESCRIPTION
**Pull request by alerickson**

### CODE REVIEW

File: tools/build.psm1 [hunk 1]
```diff
@@ -208,7 +208,7 @@ function Invoke-PowerShellGetTest {
 
     $PesterTag = '' # Conveys all test priorities
     if (-not $IsFullTestPass) {
-        $PesterTag = 'BVT' # Only BVTs
+        #$PesterTag = 'BVT' # Only BVTs
```
Comment (edyoung) [inline tools/build.psm1]: Did you mean to check in this change?

**Review by edyoung — APPROVED** [review]
Thanks!

File: tools/build.psm1 (hunk 1 above)
Comment (alerickson) [reply tools/build.psm1]: Yes, I just want to make sure all the tests (the daily tests) pass.  I'll uncomment with the last commit.

File: src/PowerShellGet/private/functions/Get-PublishLocation.ps1 [hunk 2]
```diff
@@ -8,34 +8,9 @@ function Get-PublishLocation
...
-            {
-                $tempPublishLocation = $Location + 'package/'
-            }
-
-            if($tempPublishLocation)
-            {
-                $PublishLocation = $tempPublishLocation
-            }
-        }
+        return $Location
```
Comment (bmanikm) [inline src/PowerShellGet/private/functions/Get-PublishLocation.ps1:14]: >return $Location [](start = 8, length = 16)

Consider specifying this as a breaking change in the release notes as this breaks the  existing automation scripts for registering the repositories.

**Pull request merged**